canonical/jaas-dashboard
# WD-3251 - Unify panel implementations

## Done

- Unify panel implementations.
- Add top level panel tests.
- Removed unused panels.
- Remove redundant components.

## QA

- Pull code
- Run `dotrun clean && dotrun serve`
- Open http://0.0.0.0:8036/
- Check that each panel works:
  - Go to the controller list and click "Register a controller"
  - Go to the model list and click on a model and then click "Modify access"
  - On a model page click on an application and then click "Configure"
  - On the application page tick a unit and then click "Run action"

## Details

https://warthogs.atlassian.net/browse/WD-3251
